### ISSUE TITLE
Just get a low accuracy of 75.8 with resnet50 on ImageNet

### ISSUE THREAD

**Issue by mountain111** [pytorch/examples#1153]
I train resnet50  on ImageNet with GPUs=8, batchsize=256, learning-rate=0.1, epochs=90, and momentum=0.90.
The attained top1 accuracy is 75.80, lower than the reported 76.15. The gap is not marginal on the large-scale ImageNet.
Why does the difference exist?

**Comment by hefangnan** [comment]
I get 75.644 with resnet50 on ImageNet，I think this might be normal？

**Comment by Yixuan423** [comment]
It's normal.
